### ISSUE TITLE
Kodi and accessibility for blind and visually impared users

### ISSUE THREAD

**Issue by RealAmethyst** [xbmc/xbmc#25308]
### Describe the bug
Here is a clear and concise description of what the problem is:

Kodi does not have accessibility built in so that blind and visually impared users can use it properly. A good example is a screen reader. Text to speech. An addon was made for this a while back, but it stopped working in Kodi 19. It would be great having this in Kodi itself.

## Expected Behavior
Here is a clear and concise description of what was expected to happen:

Kodi should provide a method for blind and low vision users to navigate via text to speech. Either by interfacing with already running screen readers, or by providing a text to speech library.

## Actual Behavior

No speech. I and others are unable to use the program.

## Possible Fix

Implement a text to speech survice into Kodi itself. This would mean that we wouldn't require external addon development.

### To Reproduce

1. Open Kodi
2. Use Kodi and notice that there is no way for a totally blind person to know what they are doing.

### Debuglog

The debuglog can be found here:
No reason to provide one, as there's nothing to log.

### Screenshots
Here are some links or screenshots to help explain the problem:

## Additional context or screenshots (if appropriate)
Here is some additional context or explanation that might help:

I cannot use Kodi, this accessibility roadblock prevents me from doing so.

### Your Environment
Used Operating system:

 - [ ] Windows
 - [ ] 
 - Operating system version/name:Windows 10
 - Kodi version:21

*note: Once the issue is made we require you to update it with new information or Kodi versions should that be required.
Team Kodi will consider your problem report however, we will not make any promises the problem will be solved.*

**Comment by RealAmethyst** [comment]
Should be fixed now.
